int16 gui_key
int16 expandfield
int16 refbox
int16 team
int16 ref_comm
int8 flagLoc
int16 dataAll_x
int16 dataAll_y
int16 dataAll_0
int16 setCalibMode
int16 CalibAutoIn
int16 robot_comm
int16 robot_mode
int16 refbox_status
string ipaddr
int16 port
int16 state_referee
int16 modestart
int16 modeskill
int8 simulation_lock
int8 ball_set
int8 interupt